Sources/Command/CommandRunner.swift

```diff
@@ -153,9 +154,12 @@ public struct CommandRunner: CommandRunning, Sendable {
                         do {
                             for try await data in stdoutPipe.fileHandleForReading.byteStream() {
                                 continuation.yield(.standardOutput([UInt8](data)))
+                                if let output = String(data: data, encoding: .utf8) {
+                                    logger?.debug("\(output)", metadata: loggerMetadata)
+                                }
                             }
                         } catch {
-                            logger?.error("Error reading stdout: \(error)")
+                            logger?.error("Error reading stdout: \(error)", metadata: loggerMetadata)
```

Comment:
@fortmarek I ended up using metadata to pass the information of the command that's being run. The log handler can then output that information.